tests/testBasisLists.py

```diff
@@ -182,42 +184,42 @@ def testCentralRegularization(self):
         try:
             self.policyDF.set("centralRegularizationStencil", 1)
             ipDiffim.makeRegularizationMatrix(self.policyDF)
-        except Exception, e:  # stencil of 1 not allowed
+        except Exception as e:  # stencil of 1 not allowed
             pass
         else:
             self.fail()
 
         self.policyDF.set("centralRegularizationStencil", 5)
         try:
```

Comment:
I think I understand how to convert the *not* allowed stencil values into the newer syntax (replace the try/except/else clause with your example above). But I am not sure how to do the same for the weirdly double-negative exceptions for stencil values that *are* allowed, and the similar cases that follow. Pointers appreciated!